src/state/is-within-visible-bounds-of-droppable.js

```diff
@@ -1,17 +1,35 @@
 // @flow
 import isWithin from './is-within';
+import { subtract, offsetSpacing } from './position';
 import type {
   Position,
   DraggableDimension,
   DroppableDimension,
   DimensionFragment,
+  Spacing,
 } from '../types';
 
 export const isPointWithin = (droppable: DroppableDimension) => {
-  const { top, right, bottom, left } = droppable.page.withMargin;
+  // Calculate the mid-drag scroll ∆ of the scroll container
+  const scroll = droppable.container.scroll;
```

Comment:
scroll: Position

Reply:
scroll actually has this shape:
```javascript
{
  current: Position,
  initial: Position,
}
```
I think it's OK for this not to be annotated because there are no moving parts here - I'm just pulling the prop off the `droppable` to make the code more readable. We're already checking the type of `droppable` when it comes in so there's no chance of anything going wrong here. Same goes for `containerBounds` below.